Find a functionally equivalent code:android.view.textservice.TextInfo.getText:COMMENT
Method Modifier: public      
Comment:/**
 * @return the text which is an input of a text service
 */

@@ -1,3 +1,6 @@
 {
-    return mText;
+    if (mCharSequence == null) {
+        return null;
+    }
+    return mCharSequence.toString();
 }

